test/suite/test_base01.py

```diff
@@ -66,7 +66,7 @@ def test_error(self):
                 gotException = True
                 self.pr('got expected exception: ' + str(e))
                 self.assertTrue(str(e).find('nvalid argument') >= 0)
-        self.assertTrue(gotException, 'expected exception')
+        self.assertTrue(gotException)
```

Comment:
`assertTrue` actually takes an optional second argument (see https://docs.python.org/3/library/unittest.html), that was the intention here.  To make it clearer, the code could be changed to:
```
  self.assertTrue(gotException, msg='expected exception')
```

Reply:
I see.